src/components/VmActions/index.js

```diff
@@ -249,16 +250,22 @@ class VmActions extends React.Component {
         id: `${idPrefix}-button-console`,
         items: consoles,
       },
-      {
+    ]
+
+    if (canService(vm.get('service'))) {
+      const service = vm.get('service').get('proto') + '://' +
+        vm.get('fqdn') + ':' + vm.get('service').get('port')
+      actions.push({
         priority: 0,
         actionDisabled: isPool || !canConsole(status),
-        shortTitle: msg.https(),
+        shortTitle: vm.get('service').get('name'),
         tooltip: msg.httpsService(),
```

Comment:
maybe add the name to the tooltip too?

Reply:
I'm not sure. I'm happy to do it, but it feels a bit like using the word you're defining in the definition. What do you think?